README.rst

```diff
@@ -43,8 +42,28 @@ so, `Open edX Installation Options`_ explains your options.
 .. _Open edX Developer Stack: https://github.com/openedx/devstack
 .. _Open edX Installation Options:  https://openedx.atlassian.net/wiki/spaces/OpenOPS/pages/60227779/Open+edX+Installation+Options
 
+Dependencies
+============
+
+In order to build and run this code you'll need the following available on your
+system:
+
+Interperters/Tools:
+
+* Python 3.8
```

Comment:
I wasn't sure that there was a canonical place that we could use as a source of truth.  Lot's of places say python 3.8 but it's not clear that any of them should serve as the source of truth.  Let me know if you have a better idea, otherwise we'll update this when we see that it's out of date for now.

Reply:
Sounds good. Maybe this could be worked out as part of the upgrade process runbooks? Feel free to mark resolved.

Reply:
+1 for adding this to the update process runbooks, if such runbooks exist.

As for sources of truth to link to:
* the most complete source-of-truth I can think of is [Tutor's config defaults](https://github.com/overhangio/tutor/blob/master/tutor/templates/config/defaults.yml). That's what determines what gets run for most/all users of the latest named release.
* if we don't want to assume Tutor, though, then the next best thing would be whatever we use to [run unit tests](https://github.com/openedx/edx-platform/blob/a4be3a17eb76842e351f101bc5c2a4acb24c676f/.github/workflows/unit-tests.yml#L15-L17), run migration checks, etc.

As @feanil is making it more and more feasible to run edx-platform without an external runner (like Tutor or Devstack), I could see *these instructions* actually becoming the source of truth, with things like unit tests and Tutor using them as a reference. I don't think we're there yet, but I like moving in a direction where edx-platform is its own independently-usable thing.

@feanil , I don't want to disrupt this work you've been doing lately since (a) the standalone-edx-platform idea is still pretty theoretical and (b) everything you're doing is generally useful, common-sense stuff.... but at some point of seriousness I think that a ADR for the standalone-edx-platform effort would be really valuable.